cluster/v1/types.go

```diff
@@ -234,3 +234,18 @@ const (
 	// ClusterNameLabelKey is the key of a label to set ManagedCluster name.
 	ClusterNameLabelKey = "open-cluster-management.io/cluster-name"
 )
+
+const (
+	// ClusterImageRegistriesAnnotationKey is an annotation key on ManagedCluster to configure image override for addons
```

Comment:
I think we should also  mention addonDeploymentConfig will override this.

Reply:
Fixed